src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -9,8 +9,11 @@
     public static final Prefix PREFIX_NAME = new Prefix("n/");
     public static final Prefix PREFIX_PHONE = new Prefix("p/");
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
-    // public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_SOCIAL_MEDIA = new Prefix("d/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
+    public static final Prefix PREFIX_LINK = new Prefix("l/");
+    public static final Prefix PREFIX_STARTTIME = new Prefix("s/");
+    public static final Prefix PREFIX_DURATION = new Prefix("d/");
```

Comment:
Nope, having the same prefixes for both duration and social media won't cause any issues because the parsers for AddressBook and MeetingsBook are separate.